build.gradle

```diff
@@ -269,4 +269,8 @@ ext.lib = [
         powermock_module_testng: "org.powermock:powermock-module-testng:1.6.5",
         powermock_api_mockito: "org.powermock:powermock-api-mockito:1.6.5",
 
+        //Jersey
```

Comment:
Sorry Edward, I am a perfectionist :-)
Please add a space between the `//` and the word `Jersey`.